validation/validate_dc2_run2.2i_object_table.ipynb

```diff
@@ -0,0 +1,1367 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Now we plot the *normalized* i-band magnitude distributions in Run 2.2i and HSC.\n",
+    "# They are normalized so we can focus on the shape of the distribution.\n",
+    "# However, the legend indicates the total number density of galaxies selected with our magnitude cut,\n",
+    "# which lets us find issues with the overall number density matching (or not).\n",
+    "\n",
+    "max_mag_i = 27\n",
+    "plt.figure(figsize=(8, 8))\n",
+    "nbins = 50\n",
+    "mag_range = [20, max_mag_i]\n",
+    "data_to_plot = df.loc[galaxy_idx, 'mag_i']\n",
+    "labels_to_plot = [\n",
+    "    f'Run 2.2i object catalog: {num_den_dc2.value:.1f} {num_den_dc2.unit:fits}',\n",
+    "]\n",
+    "plt.hist(data_to_plot, nbins, range=mag_range, histtype='step',\n",
+    "         label=labels_to_plot, linewidth=2.0, density=True)\n",
+    "    \n",
+    "plt.legend(loc='upper left')\n",
+    "plt.xlabel('i-band magnitude')\n",
+    "plt.ylabel('normalized distribution')\n",
+    "plt.yscale('log')\n",
+    "plt.savefig('dc2_object_run2.2i_galaxy_counts.pdf')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Magnitude Error vs. Magnitude"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The magnitude uncertainties come directly from the poisson estimates of the flux measurements.  By construction they will follow smooth curves.  We here confirm that they do."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_mag_magerr(df, band, ax=None, range=(16, 28), magerr_limit=0.25, vmin=100):\n",
+    "    # Restrict to reasonable range\n",
+    "    mag_col, magerr_col = f'mag_{band}', f'magerr_{band}'\n",
+    "    good = df[df[magerr_col] < magerr_limit]\n",
+    "\n",
+    "    ax.hexbin(good[mag_col], good[magerr_col], vmin=vmin)\n",
+    "    ax.set_xlabel(band)\n",
+    "    ax.set_ylabel(f'{band} err');\n",
+    "    ax.set_ylim(0, magerr_limit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(2, 3, figsize=(12, 6))\n",
+    "for ax, filt in zip(axes.flat, filters):\n",
+    "    mag_col, magerr_col = f'mag_{filt}', f'magerr_{filt}'\n",
+    "    plot_mag_magerr(df.loc[galaxy_idx, [mag_col, magerr_col]], filt, ax=ax)\n",
+    "plt.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(2, 3, figsize=(12, 6))\n",
+    "for ax, filt in zip(axes.flat, filters):\n",
+    "    mag_col, magerr_col = f'mag_{filt}', f'magerr_{filt}'\n",
+    "    plot_mag_magerr(df.loc[star_idx, [mag_col, magerr_col]], filt, ax=ax)\n",
```

Comment:
`vmin` argument for `plot_mag_magerr` is default to 100, which may or may not work depending on how many tracts one loads. Add a comments about changing `vmin` is an error occurs.